translate.py

```diff
@@ -3,22 +3,39 @@
 
 from __future__ import unicode_literals
 import configargparse
-
+import codecs
+from itertools import islice
 from onmt.utils.logging import init_logger
 from onmt.translate.translator import build_translator
 
 import onmt.opts as opts
 
 
+def split_corpus(path, shard_size):
```

Comment:
Is there a way to do this without copy-pasting from preprocess.py?

Reply:
yes, I'll move this to utils/misc.py